src/serac/physics/utilities/functional/detail/metaprogramming.hpp

```diff
@@ -118,3 +118,82 @@ inline constexpr void for_constexpr(lambda&& f)
 {
   detail::for_constexpr(f, std::make_integer_sequence<int, n>{}...);
 }
+
+// sam's templates for undoing for_constexpr for faster compile times
+template <int n1, typename lambda>
+void for_loop(lambda f)
```

Comment:
we may want to consider passing the lambda by rvalue reference, but I'm not 100% sure when that is preferable